src/actions/LoginActions.tsx

```diff
@@ -321,13 +321,13 @@ async function safeCreateWallet(account: EdgeAccount, walletType: string, wallet
         data: { currencyCode: wallet.currencyInfo.currencyCode, walletId: wallet.id }
       })
     }
-    dispatch(trackAccountEvent('Signup_Wallets_Created_Success'))
+    dispatch(trackAccountEvent('Signup_Wallets_Created_Success')).catch(err => showError(err))
```

Comment:
I'll look into this one more. Question is whether it should be non-async and log it's own errors like `log`, or is it different because it's revenue related and we want to block on this async call.